libs/features/mas/src/lana.js

```diff
@@ -61,26 +64,36 @@ const lanaAppender = {
         });
 
         if (errors.length) {
-            payload += ' ';
-            payload += errors.map(serializeValue).join(' ');
+            payload += ' ' + errors.map(serializeValue).join(' ');
         }
 
-        // TODO: Lana backend extracts referer from the header sent,
-        // sometimes it includes page path and query, sometimes - not
         const { pathname, search } = window.location;
-        payload += `${delimiter}page=`;
-        payload += pathname + search;
+        payload += `${config.delimiter}page=${pathname}${search}`;
 
         if (values.length) {
-            payload += `${delimiter}facts=`;
+            payload += `${config.delimiter}facts=`;
             payload += JSON.stringify(values, serializeParam);
         }
 
         if (!seenPayloads.has(payload)) {
             seenPayloads.add(payload);
-            window.lana?.log(payload, { sampleRate, tags, clientId });
+            window.lana?.log(payload, config);
```

Comment:
Nit: Check if NALA is not available (undefined) to prevent silent failures:
```suggestion
 if (window.lana && typeof window.lana.log === 'function') {
    window.lana.log(payload, config);
} else {
    console.warn('Lana logging is not available.');
    // Optionally, implement a fallback logging mechanism here
}
```

Reply:
I think it is not good idea to log `Lana logging is not available` continuously.